docs/changelog/139798.yaml

```diff
@@ -0,0 +1,6 @@
+pr: 139798
+summary: Fix interpolation for data points at bucket boundaries
+area: ES|QL
+type: bug
```

Comment:
I think we can make this issue an non-issue, given that this bug isn't released?